spectrum-offchain-lm/src/data/order.rs

```diff
@@ -393,7 +396,6 @@ impl TryFromBox for Deposit {
 #[serde(into = "RawRedeem")]
 pub struct Redeem {
     pub order_id: OrderId,
-    pub pool_id: PoolId,
```

Comment:
Pool id used to be in the redeem contract, in an older version. It no longer resides there.